tests/cli/fixtures/build/serverless.yml

```diff
@@ -128,7 +128,7 @@ functions:
               - Fn::GetAtt:
                   - StateEventTimestreamTable
                   - Arn
-              - Ref: AWS::NoValue
+              - arn:aws:timestream:*:*:database/nonexistent/table/nonexistent
```

Comment:
Similar to above, do you want to remove the references to `TimestreamIsSupportedInRegion` in the fixture?

Reply:
we want to keep that for the IAM policy stmts. We have the override the entire StateEventTimestreamDatabase and StateEventTimestreamDatabaseTable, so I removed it from them.